app/models/columns/titre_identite_column.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class Columns::TitreIdentiteColumn < Column
+  private
+
+  def get_raw_value(champ)
+    champ.piece_justificative_file.attached?.to_s
```

Comment:
On ne peut pas encore retourner un boolean ?

```suggestion
    champ.piece_justificative_file.attached?.to_s
```

Reply:
Pas ici. Mais je vais regarder au moment de faire le renommage `get_value` -> `value` si je peux faire mieux